demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -388,27 +391,43 @@ def update_existing_rn(self, current_rn, changed_files):
                     new_rn += new_rn_part
         return new_rn
 
-    def create_markdown(self, release_notes_path: str, rn_string: str, changed_files: dict):
+    def create_markdown(self, release_notes_path: str, rn_string: str, changed_files: dict,
+                        docker_image_name: Optional[str]):
         if os.path.exists(release_notes_path) and self.update_type is not None:
```

Comment:
@ShahafBenYakir it will get into the elif or else statements (and i have tested it as well myself).
check the If above:
if os.path.exists(release_notes_path) and self.update_type is not None:
it will enter this if only if file exists and -u flag was given (which is an invalid case)